script/autotest

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
@donbonifacio oh we could have that. Usually I like to wrap those things in some script files, because I'm a bit lazy and almost all the times I don't remember all the commands.. so I tend to prefer to have a script folder with this kind of scripts